training/Plugins/CUQIpy-FEniCS/CF01_Poisson_2D_CUQIpy-FEniCS.ipynb

```diff
@@ -345,21 +353,21 @@
    "metadata": {},
    "source": [
     "### 2.1. Create domain geometry \n",
-    "We model $m$ as a Matern-class random field which lead to the parametrization (Karhunen-Lo\\'eve (KL) expansion)\n",
+    "We model $m$ as a Matern-class random field which lead to the parametrization (Karhunen-Lo\\'eve (KL) expansion):\n",
```

Comment:
- Karhunen–Loève seems better
- BTW, do you want to introduce the number of KL terms in the equation following this sentence? Otherwise, people might be curious where `num_terms=32` comes from later in the code. What do you think?

Reply:
Very good point, I illustrated more here to explain the truncation